src/webpack/presets/optimize.js

```diff
@@ -9,6 +10,9 @@ export default {
 
     return {
       plugins: [
+        // Due to an issue in Webpack, its chunkhashes aren't deterministic.
```

Comment:
No, and it might not even be needed anymore, this was just moved from another file.